<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Listening</class>
 <widget class="QWidget" name="Listening">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>70</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>620</width>
         <height>480</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>620</width>
         <height>480</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <widget class="QLabel" name="mwcMqStatusImg">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>21</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>MWC MQS listener status</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources_desktop.qrc">:/img/StatusOk@2x.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="mwcMqTriggerButton">
        <property name="geometry">
         <rect>
          <x>446</x>
          <y>10</y>
          <width>175</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Start/Stop listening for messages at MWC MQS</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="control::MwcLineEditNormal" name="mwcmqsAddress">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>620</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>You MWC MQS address</string>
        </property>
        <property name="maxLength">
         <number>1024</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="mwcMQlable">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>94</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>MWC MQS</string>
        </property>
       </widget>
       <widget class="control::MwcLabelTiny" name="mwcMqStatusTxt">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>33</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Online</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="mwcMqToIndex">
        <property name="geometry">
         <rect>
          <x>445</x>
          <y>310</y>
          <width>175</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Select a specific index for your TOR and MWC MQS address</string>
        </property>
        <property name="text">
         <string>To index</string>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="mwcMqNextAddress">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>310</y>
          <width>175</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Use the next address for listening to TOR and MWC MQS messages</string>
        </property>
        <property name="text">
         <string>Next Address</string>
        </property>
       </widget>
       <widget class="control::MwcLabelTiny" name="mwcmqsAddressIndexLabel">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>110</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgba(255,255,255, 0.7)</string>
        </property>
        <property name="text">
         <string>Address index: 4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="control::MwcLine" name="frame_3">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>380</y>
          <width>201</width>
          <height>2</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="label_http">
        <property name="geometry">
         <rect>
          <x>34</x>
          <y>400</y>
          <width>85</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Http</string>
        </property>
       </widget>
       <widget class="QLabel" name="httpStatusImg">
        <property name="geometry">
         <rect>
          <x>124</x>
          <y>410</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Foreign API listener status</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources_desktop.qrc">:/img/StatusOk@2x.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcLabelTiny" name="httpStatusTxt">
        <property name="geometry">
         <rect>
          <x>34</x>
          <y>425</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Online</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="httpConfigButton">
        <property name="geometry">
         <rect>
          <x>445</x>
          <y>410</y>
          <width>175</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Configure Wallet foreign REST API</string>
        </property>
        <property name="text">
         <string>Configure</string>
        </property>
       </widget>
       <widget class="control::MwcLabelSmall" name="http_warnings">
        <property name="geometry">
         <rect>
          <x>155</x>
          <y>385</y>
          <width>211</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcLabelTiny" name="torStatusTxt">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>165</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Online</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="torStatusImg">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>150</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Tor listener status</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources_desktop.qrc">:/img/StatusOk@2x.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>85</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Tor</string>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="torTriggerButton">
        <property name="geometry">
         <rect>
          <x>446</x>
          <y>150</y>
          <width>175</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Start/Stop Tor listener</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="control::MwcLabelTiny" name="torAddressIndexLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>245</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgba(255,255,255, 0.7)</string>
        </property>
        <property name="text">
         <string>Address index: 4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="control::MwcLineEditNormal" name="torAddress">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>205</y>
          <width>620</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>You MWC MQS address</string>
        </property>
        <property name="maxLength">
         <number>1024</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcLabelSmall" name="label">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>285</y>
          <width>511</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Please stop TOR and MWC MQS listeners in order to change the wallet addess</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control_desktop/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelTiny</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLine</class>
   <extends>QFrame</extends>
   <header>control_desktop/MwcFrameWithBorder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mwcMqTriggerButton</tabstop>
  <tabstop>mwcmqsAddress</tabstop>
  <tabstop>torTriggerButton</tabstop>
  <tabstop>torAddress</tabstop>
  <tabstop>mwcMqNextAddress</tabstop>
  <tabstop>mwcMqToIndex</tabstop>
  <tabstop>httpConfigButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources_desktop.qrc"/>
 </resources>
 <connections/>
</ui>
